src/components/list-item/list-item.tsx

```diff
@@ -143,18 +135,6 @@ export class ListItemComponent implements ListItem {
     public type: 'listitem' | 'menuitem' | 'option' | 'radio' | 'checkbox' =
         'listitem';
 
-    /**
-     * Emitted when the list item toggles selection (only for selectable types and not disabled).
-     */
-    @Event()
-    public interact: EventEmitter<{
```

Comment:
This seems like a very breaking change, and ~must~ _might have to_ be marked as such.

EDIT: I guess it depends on whether there was ever a legitimate use-case for the stuff that was removed. If not, then this can perhaps just go as a bugfix.

Reply:
> @adrianschmidt the component is `@private` and only other components internally are using it.

Ah, ok! Then it's fine, yes 😄